Noise in integration tests

Integration tests running in travis have a lot of lines like below that make it difficult to spot true issues:

```
client_integration_test.go:421: error renewing TGT: [Root cause: Networking_Error] KRBMessage_Handling_Error: Error renewing TGT < Networking_Error: TGS Exchange Error: issue sending TGS_REQ to KDC: KRB Error: (13) KDC_ERR_BADOPTION KDC cannot accommodate requested option - TICKET NOT RENEWABLE
```

--- a/testenv/docker/krb5kdc-latest/kdc.conf
+++ b/testenv/docker/krb5kdc-latest/kdc.conf
@@ -11,4 +11,5 @@
   dict_file = /usr/share/dict/words
   admin_keytab = /opt/krb5/data/kadm5.keytab
   supported_enctypes = aes128-cts-hmac-sha1-96:normal aes256-cts-hmac-sha1-96:normal aes128-cts-hmac-sha256-128:normal aes256-cts-hmac-sha384-192:normal des3-cbc-sha1-kd:normal rc4-hmac:normal
+  default_principal_flags = +renewable
  }

--- a/testenv/docker/krb5kdc-older/kdc.conf
+++ b/testenv/docker/krb5kdc-older/kdc.conf
@@ -11,4 +11,5 @@
   dict_file = /usr/share/dict/words
   admin_keytab = /opt/krb5/data/kadm5.keytab
   supported_enctypes = aes128-cts-hmac-sha1-96:normal aes256-cts-hmac-sha1-96:normal des3-cbc-sha1-kd:normal rc4-hmac:normal
+  default_principal_flags = +renewable
  }

--- a/testenv/docker/krb5kdc-res/kdc.conf
+++ b/testenv/docker/krb5kdc-res/kdc.conf
@@ -11,4 +11,6 @@
   dict_file = /usr/share/dict/words
   admin_keytab = /opt/krb5/data/kadm5.keytab
   supported_enctypes = aes128-cts-hmac-sha1-96:normal aes256-cts-hmac-sha1-96:normal aes128-cts-hmac-sha256-128:normal aes256-cts-hmac-sha384-192:normal des3-cbc-sha1-kd:normal rc4-hmac:normal
+  default_principal_flags = +renewable
  }
+

--- a/testenv/docker/krb5kdc-shorttickets/kdc.conf
+++ b/testenv/docker/krb5kdc-shorttickets/kdc.conf
@@ -11,4 +11,6 @@
   dict_file = /usr/share/dict/words
   admin_keytab = /opt/krb5/data/kadm5.keytab
   supported_enctypes = aes128-cts-hmac-sha1-96:normal aes256-cts-hmac-sha1-96:normal aes128-cts-hmac-sha256-128:normal aes256-cts-hmac-sha384-192:normal des3-cbc-sha1-kd:normal rc4-hmac:normal
+  default_principal_flags = +renewable
  }
+

--- a/testenv/docker/krb5kdc/kdc.conf
+++ b/testenv/docker/krb5kdc/kdc.conf
@@ -11,4 +11,5 @@
   dict_file = /usr/share/dict/words
   admin_keytab = /opt/krb5/data/kadm5.keytab
   supported_enctypes = aes128-cts-hmac-sha1-96:normal aes256-cts-hmac-sha1-96:normal aes128-cts-hmac-sha256-128:normal aes256-cts-hmac-sha384-192:normal des3-cbc-sha1-kd:normal rc4-hmac:normal
+  default_principal_flags = +renewable
  }

--- a/testenv/krb5kdc-vagrant/kdc.conf
+++ b/testenv/krb5kdc-vagrant/kdc.conf
@@ -11,4 +11,5 @@
   dict_file = /usr/share/dict/words
   admin_keytab = /opt/krb5/data/kadm5.keytab
   supported_enctypes = aes128-cts-hmac-sha1-96:normal aes256-cts-hmac-sha1-96:normal aes128-cts-hmac-sha256-128:normal aes256-cts-hmac-sha384-192:normal des3-cbc-sha1-kd:normal rc4-hmac:normal
+  default_principal_flags = +renewable
  }

--- a/testenv/latest-krb5kdc-vagrant/kdc.conf
+++ b/testenv/latest-krb5kdc-vagrant/kdc.conf
@@ -11,4 +11,5 @@
   dict_file = /usr/share/dict/words
   admin_keytab = /opt/krb5/data/kadm5.keytab
   supported_enctypes = aes128-cts-hmac-sha1-96:normal aes256-cts-hmac-sha1-96:normal aes128-cts-hmac-sha256-128:normal aes256-cts-hmac-sha384-192:normal des3-cbc-sha1-kd:normal rc4-hmac:normal
+  default_principal_flags = +renewable
  }

--- a/testenv/older-krb5kdc-vagrant/kdc.conf
+++ b/testenv/older-krb5kdc-vagrant/kdc.conf
@@ -11,4 +11,5 @@
   dict_file = /usr/share/dict/words
   admin_keytab = /opt/krb5/data/kadm5.keytab
   supported_enctypes = aes128-cts-hmac-sha1-96:normal aes256-cts-hmac-sha1-96:normal des3-cbc-sha1-kd:normal rc4-hmac:normal
+  default_principal_flags = +renewable
  }

